build.gradle

```diff
@@ -69,8 +69,7 @@ dependencies {
     library "org.reflections:reflections:${project.reflections_version}"
     library("io.netty:netty-handler-proxy:${project.netty_version}") { transitive = false }
     library("io.netty:netty-codec-socks:${project.netty_version}") { transitive = false }
-    library "com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"
-    annotationProcessor "com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"
+    annotationProcessor(include(implementation("com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}")))
```

Comment:
why?

Reply:
there might be some class conflict as https://github.com/RaphiMC/ImmediatelyFast/commit/d2394173c393ed7c5a1f5ef9a3945cc48f37b7a9 mentioned

Reply:
also see the document here https://gist.github.com/LlamaLad7/ec597b6d02d39b8a2e35559f9fcce42f
it has official usage